defaults.yml

```diff
@@ -60,3 +60,6 @@ auth:
     salt: secretsauce # Required when upgrading from version < 0.3
 
 noRobots: true # Serve robots.txt with disallow
+
+rooms:
+  passworded: false
```

Comment:
Why not `allowPasswords` but `privateRooms` in xmpp protocol means a room who only invited users can enter ;)

BTW in the xmpp protocol `passworded rooms` are `protected room`

Let me know what is your choose!